internal/functions/deploy/deploy.go

```diff
@@ -95,7 +98,10 @@ func makeLegacyFunctionBody(functionBody io.Reader) (string, error) {
 	return buf.String(), nil
 }
 
-func deployFunction(ctx context.Context, projectRef, slug string, functionBody io.Reader, verifyJWT, useLegacyBundle bool) error {
+func deployFunction(ctx context.Context, projectRef, slug string, functionBody io.Reader, verifyJWT, useLegacyBundle bool, functionSize int) error {
+	functionSizeFmt := fmt.Sprintf("%.2fMB", float64(functionSize)/MB)
+	fmt.Println("Deploying " + utils.Bold(slug) + " (script size: " + utils.Bold(functionSizeFmt) + ")")
```

Comment:
How about moving this print statement to line 87? then we dont need to change this function signature.